splicing method (#65)
d3b-center/OpenPedCan-methods: content/06.methods.md

Fusions were retained in this module if they were called by both callers, recurrent or specific to a cancer group, or annotated as a putative oncogenic fusion.
Please refer to the module linked above for more detailed documentation and scripts.

##### Splicing quantification

> To detect alternative splicing events, we utilized rMATS turbo (v. 4.1.0) with Ensembl/Gencode v39 GFF annotations, following the Kids First RNA-Seq workflow guidelines (https://github.com/kids-first/kf-rnaseq-workflow/blob/master/workflow/rmats_wf.cwl). We applied additional filters to include only splicing events with total junction read counts greater than 10. Subsequently, we merged the output files for each sample to create a comprehensive merged TSV file containing all splicing events across all samples.

Reviewer: ```suggestion
To detect alternative splicing events, we utilized rMATS turbo (v. 4.1.0) with Ensembl/Gencode v39 GFF annotations using the [Kids First RNA-Seq workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/master/workflow/rmats_wf.cwl). 
We applied an additional filter to include only splicing events with total junction read counts greater than 10.
```
greater than or greater than OR equal to?

we can remove merging info, as we do for all data

Can you add something here about the read length parameter/ paired param ?

Fusions were retained in this module if they were called by both callers, recurrent or specific to a cancer group, or annotated as a putative oncogenic fusion.
Please refer to the module linked above for more detailed documentation and scripts.

##### Splicing quantification

To detect alternative splicing events, we utilized rMATS turbo (v. 4.1.0) with Ensembl/GENCODE v39 GFF annotations using the [Kids First RNA-Seq workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/master/workflow/rmats_wf.cwl).

> We used --variable-read-length and -t paired options and applied an additional filter to include only splicing events with total junction read counts greater than 10.

Reviewer: ```suggestion
We used `--variable-read-length` and `-t paired` options and applied an additional filter to include only splicing events with total junction read counts greater than 10.
```
